Xamarin.Forms.Platform.UAP/CarouselView/FormsViewToNativeUWP.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+    public static class FormsViewToNativeUWP
+    {
+        public static FrameworkElement ConvertFormsToNative(Xamarin.Forms.View view, Rectangle size)
+        {
+            //var vRenderer = RendererFactory.GetRenderer (view);
```

Comment:
Could you add some brief explanation of the limitations and why the code is different for this platform? Something like "normally, we'd do X but on Windows we have to do Y because...". That way future maintainers won't waste time trying to do 'X'.